adls/include/adls_client.h

```diff
@@ -0,0 +1,93 @@
+#pragma once
+
+#ifdef _WIN32
+#ifdef azure_storage_adls_EXPORTS
+#define AZURE_STORAGE_ADLS_API __declspec(dllexport)
+#else
+#define AZURE_STORAGE_ADLS_API __declspec(dllimport)
+#endif
+#else /* ifdef _WIN32 */
+#define AZURE_STORAGE_ADLS_API
+#endif
+
+#include "blob/blob_client.h"
+#include "nlohmann_json_parser.h"
```

Comment:
>nlohmann_json_parser [](start = 10, length = 20)

Can this be moved to .cpp file?

Reply:
fixed in https://github.com/Azure/azure-storage-cpplite/pull/50/commits/829c47222865f8be7f2524a745c3f31cfa1c968b